src/main/java/com.gigssandbox/command/CommandFactory.java

```diff
@@ -3,14 +3,9 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class CommandFromStringArray {
-    private String[] strings;
+public class CommandFactory {
 
-    public CommandFromStringArray(String[] strings) {
-        this.strings = strings;
-    }
-
-    public Command value() {
+    public static Command from(String... strings) {
```

Comment:
method name should be verb. Name it "create()"

Reply:
But J. Bloch recommends to name fabric methods in the following way:
![_pjstk4wz50](https://user-images.githubusercontent.com/12383964/52080938-06c8c600-25aa-11e9-950f-37f19347f8f0.jpg)